content/ru/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Функция API Priority and Fairness
```

Comment:
Thank you, Tim! We're always challenged by these small things. From one side, translating specific terms makes them sound a bit awkward (or is it just unusual?..) in other languages. On another side, it's better to have as much text in Russian as possible; otherwise, the documentation might end up being (over)crowded with English terms. I agree that we better use the Russian translation of the term everywhere while we also specify the original name of the feature when we mention it in this document for the first time.

Reply:
The challenge doing localization is that the people doing it always understand the source language well, and then you have to imagine yourself as someone who doesn't. So long as you're happy with you're judgement and you know you've thought things through, you should go with what feels right.